build.sbt

```diff
@@ -81,7 +82,7 @@ val commonSettings = Def.settings(
   scalacOptions ++= Seq(
       "-encoding",
       "UTF-8", // yes, this is 2 args
-      "-Wconf:cat=feature:w,cat=deprecation:w,cat=unchecked:w,cat=lint:w,cat=unused:w,cat=w-flag:w"
+      "-Wconf:cat=feature:w,cat=deprecation&msg=.*JavaConverters.*:s,cat=unchecked:w,cat=lint:w,cat=unused:w,cat=w-flag:w"
```

Comment:
to avoid all warnings for JavaConverters, but not silence everything
https://www.scala-lang.org/2021/01/12/configuring-and-suppressing-warnings.html